apps/condo/domains/resident/constants/constants.js

```diff
@@ -100,7 +100,7 @@ const PAYMENT_CATEGORIES_META = [
     },
     {
         id: '7',
-        name: 'ТКО',
+        name: 'Мусор',
```

Comment:
Why this are present in current task?

Reply:
because I made a commit related to another task. I don't see any problem with this
[fix(condo): DOMA-7567 change payment categories translations](https://github.com/open-condo-software/condo/pull/4487/commits/9689341512686e0624a5ffc36de235fdb5445b4b)

Reply:
It's a bit unclear. Good practice is 1 task = 1 PR, so it's easy to find changes after squash